### DISCUSSION TITLE
Malacocincla perspicillata sighting is not a specimen

### DISCUSSION THREAD

**Discussion by mdoering** [plazi/community#80] [Ideas]
The first observation incl photo of Malacocincla perspicillata for 170 years in 2020 is not a preserved specimen.
Please correct `https://dwc.tdwg.org/terms/#dwc:basisOfRecord` to be HumanObservation.

**Context**
[GBIF occurrence](https://www.gbif.org/occurrence/3042816301)
[Plazi reference](http://treatment.plazi.org/id/03C887B0FF80FFD4FFE9FE21D125FD77)

**Comment by myrmoteras** [comment]
we only have materialsCitations that turn into occurrences in GBIF, an issue known on both sides.

Not sure, whether we can fix this on our side right now.

**Comment by mdoering** [comment]
I assume you just mark all material citations as PreservedSpecimen in Darwin Core. But that example proves this is not necessarily correct - even though the vast majority must be. How about allowing to configure the BoR for an entire dataset and default to PreservedSpecimen? Then you could at least manually switch to Occurrence, HumanObservation or MachineObservation?

**Comment by myrmoteras** [comment]
Will explore and get back to you.

Today's new species: http://tb.plazi.org/GgServer/static/newToday.html

________________________________

**Comment by mdoering** [comment]
Thanks. I can imagine for new species descriptions it is also more common to not kill the subject.

**Comment by myrmoteras** [comment]
I guess this not killing is not reality yet and has a huge resistance in the community.
